### DISCUSSION TITLE
wp-config.php missing configuration

### DISCUSSION THREAD

**Discussion by damon-t** [evertramos/docker-wordpress#56] [Q&A]
Hi,

after running a fresh container the wp-config.php file is not set with the right data from the .env file. with the previous version it was working.
Could you please take a look on this?

**Comment by evertramos** [comment]
what is missing?

**Comment by damon-t** [comment]
I just ran into an error while running wpcli.

actually I thought that the MySQL settings in wpconfig.php will be get from the .env file. So I don't need to paste the same data again in a different file, or do I have to? 
Sorry, If it's confusing, do you know what I mean?

**Comment by evertramos** [comment]
no I don't... you must use wp cli as of:

```bash
docker-compose run --rm wpcli
```

please check:
```bash
./wp-cli-test.sh
```

**Comment by damon-t** [comment]
sure, I am running wp-cli like you mentioned. 
Do I have to configure wp-config.php manually before I can run wp-cli?

**Reply by evertramos** [reply]
no you don't

**Reply by damon-t** [reply]
In my case after I configure the .env file and run "docker-compose up -d" the wp-config.php file lookes like this:

 ```
42 // ** MySQL settings - You can get this info from your web host ** //
 43 /** The name of the database for WordPress */
 44 define( 'DB_NAME', getenv_docker('WORDPRESS_DB_NAME', 'wordpress') );
 45
 46 /** MySQL database username */
 47 define( 'DB_USER', getenv_docker('WORDPRESS_DB_USER', 'example username') );
 48
 49 /** MySQL database password */
 50 define( 'DB_PASSWORD', getenv_docker('WORDPRESS_DB_PASSWORD', 'example password') );

```

and then when I want to run "docker-compose run --rm wpcli" I get an error.
"Error: Error establishing a database connection."

**Reply by evertramos** [reply]
The wp-config.php file are created when you start the container for the first time, it does not update when you change the .env file... if you change the env file you must change the wp-config.php you are mixing up responsibilities. This has nothing to do with this repo in fact but the Wordpress container.
